src/connecpy/client.py

```diff
@@ -21,17 +22,25 @@ class ConnecpyClient:
 
     Args:
         address (str): The address of the Connecpy server.
-        session (httpx.Client): The httpx client session to use for making requests.
+        timeout_ms (int): The timeout in ms for the overall request. Note, this is currently only implemented
```

Comment:
In unix, it's not hard to do better than read timeout but I don't know if it's worth improving this for only sync client. Can reexamine later when supporting streaming

Reply:
I see. Makes sense 👍